.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - openssl1.1
+  pull_request:
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    strategy:
```

Comment:
Can we disable fast-failure so that we'll get all results even if one of the jobs fails?

Reply:
I've kept it like this on purpose, just to be economical with CPU time. But if you prefer it that way, I'll change it. :+1: